projects/origin-release/golang-1.12/Dockerfile

```diff
@@ -28,7 +28,7 @@ RUN yum install -y epel-release && \
 ADD *.rpm /tmp/origin-rpm/
 RUN yum install -y /tmp/origin-rpm/*.rpm && \
     yum clean all && \
-    curl https://storage.googleapis.com/golang/go$VERSION.linux-amd64.tar.gz | tar -C /usr/local -xzf - && \
+    curl https://storage.googleapis.com/golang/go$VERSION.linux-$(uname -m | sed 's/x86_64/amd64/g').tar.gz | tar -C /usr/local -xzf - && \
```

Comment:
In line no. 44 also we need to remove reference for amd64 for dep command download

Reply:
Updated.